app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { getConfig_nimbusConfig } from "../../types/getConfig";
+import SummaryTimeline from "../SummaryTimeline";
+import TableSummary from "../TableSummary";
+import TableAudience from "../TableAudience";
+import LinkExternal from "../LinkExternal";
+import { NimbusExperimentStatus } from "../../types/globalTypes";
+
+type SummaryProps = {
+  experiment: getExperiment_experimentBySlug;
+};
+
+const Summary = ({ experiment }: SummaryProps) => (
+  <div data-testid="summary">
+    <h2 className="h5 mb-3">Timeline</h2>
+    <SummaryTimeline {...{ experiment }} />
+
+    <div className="d-flex flex-row justify-content-between">
+      <h2 className="h5 mb-3">Summary</h2>
+      {experiment.status !== NimbusExperimentStatus.DRAFT && (
+        <span>
+          <LinkExternal
+            href={`https://experimenter.services.mozilla.com/api/v6/experiments/${experiment.slug}/`}
```

Comment:
Is v6 correct? I've been using v3 in the analysis hook, based on Rapid, but maybe that's not right anymore

Reply:
That one hits the visualization endpoint which differs from this one.

However, I realize this shouldn't have `https://experimenter.services.mozilla.com`, it'd reflect prod link locally and on stage. Doh.